svm/src/account_loader.rs

```diff
@@ -95,6 +96,264 @@ pub struct FeesOnlyTransaction {
     pub fee_details: FeeDetails,
 }
 
+#[derive(Debug, Clone)]
```

Comment:
afaict `Clone` is only used for tests.

Can we either refactor tests to not do that, or only derive this for tests?

(also remove `Debug` unless you need it)

Reply:
yep will do something with `Clone`, its never used outside of tests. is there a reason to not always derive `Debug` tho? i normally put it on every single struct/enum because it makes debugging test failures easy in the future

Reply:
`Debug` is not terrible, the main annoyance I find with it is that it "infects" everything it touches. Suddenly everything inside this struct now also needs to be `Debug` as well. Which can be annoying if somewhere down the line we refactor some internal field and it does not implement `Debug`. Now I either have to remove it all the way up the structure chains, or implement debug manually.

Reply:
i was gonna keep it but actually i will remove it, i just found out while testing something that both mock bank callback structs in the two files unit tests dont derive `Debug`, but because `AccountLoader` goes through a trait, you dont find out that your derive isnt usable until you actually try to use it